components/navbar/NavbarExploreOptions.vue

```diff
@@ -15,30 +15,31 @@
       </span>
     </span>
     <hr aria-role="menuitem" class="dropdown-divider my-4" />
-    <span
-      v-for="option in availableChains.slice(0, 3)"
-      :key="option.value"
-      :class="[
-        'menu-item',
-        'mr-2',
-        { 'is-active': selectedChain === option.value },
-      ]"
-      :value="option.value"
-      @click="setSelectedChain(option.value)">
-      {{ option.text }}
-    </span>
+    <div>
+      <span
+        v-for="option in filteredChains"
+        :key="option.value"
+        :class="['menu-item', 'mr-2']"
+        :value="option.value"
+        @click="setSelectedChain(option.value)">
+        {{ option.text }}
+      </span>
+    </div>
   </div>
 </template>
 
 <script lang="ts" setup>
-const { $store, $router } = useNuxtApp()
+const { $router } = useNuxtApp()
 const { urlPrefix } = usePrefix()
 const { availableChains } = useChain()
 
-const selectedChain = $store.getters.getSettings['urlPrefix']
+const filteredChains = computed(() => {
+  return availableChains?.value.filter((chain) => {
+    return ['rmrk2', 'bsx', 'rmrk'].includes(chain.value)
+  })
+})
 
 const setSelectedChain = (value) => {
-  $store.dispatch('setUrlPrefix', value)
```

Comment:
it could be a problem if you remove this line

Reply:
why? i thought we werent changing chains from here?

Reply:
Hmm.. I may miss something but how would you change chain then?

Reply:
From `ChainSelectDropdown`? I would like to see whatt happen if we don't change chain from this navbarExploreOptions. 
(netlify preview failed)

Reply:
I think chains are only suppose to change from here now
<img width="268" alt="Screen Shot 2023-04-05 at 11 39 48 PM" src="https://user-images.githubusercontent.com/39299315/230292130-405ae6e8-bb7e-43be-a070-16b86ebb40fa.png">


@exezbcz can you confirm

Reply:
yes, that how it should be. I think there was a lot of confusion around the explore dropdown chains - they are only shortcuts to explore on that chain (same as the ones under search bar)